packages/studio-plugin/tests/sourcefiles/PageFile.getPageState.test.ts

```diff
@@ -69,13 +73,14 @@ describe("getPageState", () => {
     ]);
   });
 
-  it("correctly parses CSS imports", () => {
+  it("correctly parses CSS  imports", () => {
     const pageFile = createPageFile("entityTemplate");
     const result = pageFile.getPageState();
 
     assertIsOk(result);
+    const expectedCssPath = getFixturePath("PageFile/index.css");
```

Comment:
nit: this variable name is a little misleading now that we expect multiple CSS paths again

Reply:
changed to `expectedIndexCssPath`!